coderd/tailnet.go

```diff
@@ -97,6 +98,18 @@ func NewServerTailnet(
 		agentConnectionTimes: map[uuid.UUID]time.Time{},
 		agentTickets:         map[uuid.UUID]map[uuid.UUID]struct{}{},
 		transport:            tailnetTransport.Clone(),
+		connsPerAgent: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: "coder",
+			Subsystem: "servertailnet",
+			Name:      "open_conns",
+			Help:      "Total number of TCP connections currently open to workspace agents.",
+		}, []string{"agent_id"}),
+		totalConns: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: "coder",
+			Subsystem: "servertailnet",
+			Name:      "total_conns",
```

Comment:
Prometheus convention is to put "total" at the end.


```suggestion
			Name:      "tcp_connections_total",
```